drop_service/notify.py

```diff
@@ -0,0 +1,70 @@
+import base64
+import logging
+from functools import lru_cache
+
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+from ws4redis.publisher import RedisPublisher
+from ws4redis.redis_store import RedisMessage
+
+from pyfcm import FCMNotification
+from pyfcm.errors import AuthenticationError, FCMServerError, InvalidDataError, InternalPackageError
+
+from .monitoring import FCM_CALLS, FCM_DURATION, monitor_duration
+from .util import CsrfExemptView
+
+
+logger = logging.getLogger('drop_service.notify')
+
+
+def get_notificators():
+    """Return list of configured notificator instances."""
+    notificators = []
+    for class_path in settings.PUSH_NOTIFICATORS:
+        cls = import_string(class_path)
+        notificators += cls(),
+    return notificators
+
+
+class FCM:
+    """
+    Publish drops through FCM topic messages. The topic is the drop ID.
+    """
+    SERVICE = 'fcm'
+    _logger = logger.getChild('fcm')
+
+    def __init__(self):
+        self._push = FCMNotification(api_key=settings.FCM_API_KEY, proxy_dict=settings.FCM_PROXY)
```

Comment:
Inject FCMNotification in the constructor (maybe with a default value that acts as a fallback) to test this class